storage-node/src/services/webApi/app.ts

```diff
@@ -186,3 +202,27 @@ function verifyUploadTokenData(accountAddress: string, token: UploadToken, reque
     throw new Error('Nonce not found')
   }
 }
+
+async function validateUploadFileParams(req: express.Request, res: express.Response<unknown, AppConfig>) {
+  const { api, queryNodeEndpoint, workerId } = res.locals
+
+  const storageBucketId = Number(req.query.storageBucketId)
+  const dataObjectId = Number(req.query.dataObjectId)
+  const bagId = req.query.bagId!.toString()
```

Comment:
It's dangerous to use non-null assertion here
```suggestion
  const bagId = req.query.bagId?.toString() || ''
```

Reply:
[aa7d7c6](https://github.com/Joystream/joystream/pull/4402/commits/aa7d7c66903226558bf95492578d3ac64eff62e6)